.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -38,3 +38,12 @@ Add as many details as possible to help others reproduce the issue and test the
 - [ ] Added changelog entry (or does not apply)
 - [ ] Covered with tests (or have a good reason not to test in description ☝️)
 - [ ] Tested on mobile (or does not apply)
+
+**Post merge**
+
+<!--
+Make sure you edit the page for the current release when adding testing instructions.
+We often create a blank page ahead of time for the next release.
+-->
+
+- [ ] Added testing instructions to the [release testing doc](https://github.com/Automattic/woocommerce-payments/wiki/Release-testing-instructions) (or does not apply)
```

Comment:
I have a suggestion that can make _what's new post_ writing easier. If the change impacts the UI somehow, then the author can add a screenshot for the same in the PR description. That can save release lead's time to try out that feature and then click the screenshot (sometimes it takes more time due to lack of context for that feature). What do you think?

Reply:
It's a good idea but I might leave it for another PR to keep this one single focus.

Also, we already have the following comment in the template:
```
<!--
Images or gifs: Include before and after screenshots or gifs/videos when it makes sense.
-->
```

I'd hope most people are including screenshots when appropriate.